ert_shared/storage/extraction_api.py

```diff
@@ -0,0 +1,276 @@
+import time
+
+from ert_data.measured import MeasuredData
+from ert_shared import ERT
+from ert_shared.feature_toggling import FeatureToggling
+from ert_shared.storage import connections
+from ert_shared.storage.blob_api import BlobApi
+from ert_shared.storage.model import ParameterPrior
+from ert_shared.storage.rdb_api import RdbApi
+
+from res.enkf.export import MisfitCollector
+
+
+def _create_ensemble(rdb_api, reference, priors):
+    if not ((reference is None) ^ (len(priors) == 0)):
+        raise ValueError("Ensembles can have only a reference or a set of priors")
+    facade = ERT.enkf_facade
+    ensemble_name = facade.get_current_case_name()
+    ensemble = rdb_api.add_ensemble(ensemble_name, reference=reference, priors=priors)
+
+    for i in range(facade.get_ensemble_size()):
+        rdb_api.add_realization(index=i, ensemble_name=ensemble.name)
+
+    return ensemble
+
+
+def _extract_and_dump_observations(rdb_api, blob_api):
+    facade = ERT.enkf_facade
+
+    observation_keys = [
+        facade.get_observation_key(nr) for nr, _ in enumerate(facade.get_observations())
+    ]
+
+    measured_data = MeasuredData(facade, observation_keys)
+    # TODO: Should save all info and info about deactivation
```

Comment:
This should either be fixed or persisted in an issue

Reply:
Already fixed, just remove TODO